Data/CosmicHappenings.json

```diff
@@ -0,0 +1,181 @@
+{
+  "category_icons": {
+    "Asteroid": "category_icon_Asteroid.png",
+    "Constellation": "category_icon_Constellation.png",
+    "Entity": "category_icon_Entity.png",
+    "Major": "category_icon_Major.png",
+    "Planet Phase": "category_icon_PlanetPhase.png",
+    "Toll of Seasons": "category_icon_TollOfSeasons.png",
+    "Weather": "category_icon_Weather.png"
+  },
+  "happenings": [
+    {
+      "name": "Astral Eclipse",
+      "type": "Minor",
+      "category": "Constellation",
```

Comment:
You can use `category` to map to the `category_icons` map to get the icon for the category.